kolibri/plugins/device/assets/src/modules/deviceInfo/index.js

```diff
@@ -23,7 +23,7 @@ export default {
   actions: {
     updateDeviceName(store, name) {
       return client({
-        path: urls['kolibri:core:devicename'](),
+        url: urls['kolibri:core:devicename'](),
         method: 'PATCH',
```

Comment:
On the axios docs, they use lower case for the method values. Does this still work and is axios case-sensitive?

```suggestion
        method: 'patch',
```

Reply:
I think it might be nicer to use the methods axios provides just to make it a little more transparent e.g. `client.patch(...)`

Reply:
It doesn't appear to be case sensitive - all the requests I tested were upper case, and all worked.

I would like us to eventually transition to just using the Axios methods directly, but wanted to limit the possibility of breakage first, then do a more aggressive refactor later.